homework-submission/react-hyf/react-hyf-client/src/Components/fetchdata.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+
+function DataFetching(source) {

```

Comment:
Nice that you abstracted data fetching in a reusable way :-)

Reply:
:3